### PR TITLE
Bump current release to 1.8 and update history page

### PR DESCRIPTION
**Pull request by carolynhu**

### CODE REVIEW

File: perf_dashboard/history/templates/history.html
Comment (ericvn) [inline perf_dashboard/history/templates/history.html:68]: For consistency, and I'n not sure when/if the archive site will go away, the older <=1.5 should be changed to use istio.io/v1.x.

**Review by ericvn — APPROVED** [review]

**Comment by carolynhu** [comment]
@ericvn thanks for your approval, I'm putting this PR on hold, since i'm fixing a UI bug, after that PR merged in, I will remove the do not merge label. Thank you!

**Comment by ericvn** [comment]
No problem @carolynhu. I figured you remove the DNM when you needed it in.

**Pull request merged**
